packages/cwd-pre-propose-base/src/error.rs

```diff
@@ -15,6 +19,15 @@ pub enum PreProposeError {
     #[error(transparent)]
     Deposit(#[from] DepositError),
 
+    #[error(transparent)]
+    Hooks(#[from] HookError),
+
+    #[error(transparent)]
+    ParseReplyError(#[from] ParseReplyError),
+
+    #[error(transparent)]
+    ParseIntError(#[from] ParseIntError),
```

Comment:
these should probably be in the implementers error enum. the base contract doesn't actually do any number parsing i don't think?

Reply:
Ideally we wouldn't have to parse the Int in a submessage. Open to commits on this.